PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+# Assignee Section
+
+## Description
+[FILL]
```

Comment:
Instead of writing `FILL` I think it should be more clearly made to people with something like `<Write a description/summary of the changes for this pull request>`, I would maybe also add an encouragment of adding a quick list of main points.